src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/VideoStream/VideoStream.view.vue

```diff
@@ -119,7 +154,7 @@ export default {
     },
 
     async openStream() {
-      if (this.stream) {
+      if (this.stream && this.stream.active) {
```

Comment:
what is the reason for check active stream?

Reply:
described above